be/src/column/nullable_column.h

```diff
@@ -198,6 +198,7 @@ class NullableColumn final : public ColumnFactory<Column, NullableColumn> {
 
     ColumnPtr& data_column() { return _data_column; }
 
+    const NullColumn& null_column_ref() const { return *_null_column; }
```

Comment:
null_column_ref is similar to ColumnRef, suggest to use another name.

Reply:
This is for consistency with here: https://github.com/StarRocks/starrocks/blob/5ecee061bda8d15009e0a4f275522fb21af87f97/be/src/column/nullable_column.h#L195

Reply:
any suggestion ?